profile/README.md

```diff
@@ -0,0 +1,33 @@
+_Hexagonal architecture for API integrations_
+
+[![NPM](https://img.shields.io/badge/NPM-%40superfaceai-4338CA)](https://www.npmjs.com/org/superfaceai)
+[![Twitter](https://img.shields.io/badge/Twitter-%40superfaceai-4338CA)](https://twitter.com/superfaceai)
+[![Discord](https://img.shields.io/badge/Discord-%40superface.ai-4338CA)](https://sfc.is/discord)
```

Comment:
@tjerabek Can you help me with right color please? 😇 

<img width="434" alt="image" src="https://user-images.githubusercontent.com/959390/154941977-2cad80dd-58c6-45a5-b63d-47d2331b94fd.png">

Reply:
@jnv what is convention for Discord? Should it be with `@`?

Reply:
@freaz No, server names are not unique and we are not boosted for vanity name. So drop the @ please.